score-board/src/Components/Counter.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+
+export class Counter extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      balls: 0,
+      strikes: 0
+    };
+  }
+
+  incrementBalls = () => {
+    this.setState({
+      balls: this.state.balls + 1
+    });
+  };
+
+  incrementStrikes = () => {
+    this.setState({
+      strikes: this.state.strikes + 1
+    });
+  };
+
```

Comment:
Try getting in the habit of grabbing prevState in your setState function i.e.
`this.setState(prevState => {
    strikes: prevState.strikes + 1
})`